### ISSUE TITLE
Doesn't install godoc

### ISSUE THREAD

**Issue by leebrooks0** [moovweb/gvm#45]
I am brand new to Go (installed it today...), and I see that GVM doesn't install other utilities such as godoc. How do I install them? If I have do an apt-get install golang-stable to get them, then how does GVM help?

**Comment by remy-actual** [comment]
Makes sense to install godoc once under the system version, since it's a database, not a versioned package and is therefore immune to the need to be managed as a build dependency. 
Installing under system $GOPATH makes godoc command universally available for reference during local development, regardless of the active gvm-managed go version.  
Above could be default behavior when running gvm-installer script, with a command flag option added to install without godoc.  
Also for reference for future googlers, current install path is 
```
$go get golang.org/x/tools/cmd/godoc
```
ref [New import paths for Go sub-respositories](http://grokbase.com/t/gg/golang-nuts/14b5zdma3z/go-nuts-new-import-paths-for-go-sub-respositories)

**Comment by BenKnigge** [comment]
I will likely close this issue in the near future unless I hear otherwise.

**Comment by rainydew** [comment]
I met the same issue. Since I use Go1.18 under GVM to write generics codes, my system go binary is go1.13.15. The Godoc is for system go binary and cannot support new generics functions.
I didn't find a way to use godoc under GVM specific version.
